docs/shared/tutorials/angular-monorepo.md

```diff
@@ -1241,50 +1241,48 @@ This tutorial walked you through how Nx can improve the local development experi
 - Nx Agents [efficiently distribute tasks across machines](/ci/concepts/parallelization-distribution) ensuring constant CI time regardless of the repository size. The right number of machines is allocated for each PR to ensure good performance without wasting compute.
 - Nx Atomizer [automatically splits](/ci/features/split-e2e-tasks) large e2e tests to distribute them across machines. Nx can also automatically [identify and rerun flaky e2e tests](/ci/features/flaky-tasks).
 
-### Generate a CI Workflow
-
-If you are starting a new project, you can use the following command to generate a CI workflow file.
+### Connect to Nx Cloud
 
-```shell
-npx nx generate ci-workflow --ci=github
-```
+Nx Cloud is a companion app for your CI system that provides remote caching, task distribution, e2e tests deflaking, better DX and more.
 
-{% callout type="note" title="Choose your CI provider" %}
-You can choose `github`, `circleci`, `azure`, `bitbucket-pipelines`, or `gitlab` for the `ci` flag.
+{% callout type="note" title="Commit and push your changes" %}
+Now that we're working on the CI pipeline, it is important for your changes to be pushed to a github repository. Commit all your changes so far and push them up to a hosted repository.
```

Comment:
We should specifically instruct them to go to https://github.com/new and create a new repo + push their main branch.